### ISSUE TITLE
Owncloud no longer compatible with Loxberry 3.0

### ISSUE THREAD

**Issue by Blaubart** [mschlenstedt/LoxBerry-Plugin-Owncloud#1]
I updated my Loxberry to version 3.0.0.6, but ownCloud dosen't work anymore. I get:
This version of ownCloud is not compatible with PHP 7.4
You are currently running PHP 7.4.33.​

**Comment by mschlenstedt** [comment]
Yes, as mentioned in the error message: The Owncload Installation is out of date. You can update Owncloud (I tried that in the past with Nextcloud - it will take you weeks....) or just install a fresh installation.

But I am not sure if this plugin still works at all - I moved to nextcloud years ago....

**Comment by Blaubart** [comment]
Thanks for your answer. But I don't really understand your answer. How should a new installation be carried out, whether with Nextcloud or owncloud? The plugin no longer works under 3.0.0.6?

**Comment by mschlenstedt** [comment]
The Plugin just installs Owncloud. If your OwnCloud installation is now out of date, you can upgrade it (with the Ownclod tools) or deinstall the plugin and try to reinstall the plugin (in the hope it installs a newer version of owncloud). I don't now if this works - the plugin is really out of date and I haven't tested it for years... The last update to this plugin was done 5 years ago!

Try it out: deinstall the plugin and reinstall it.

**Comment by mschlenstedt** [comment]
Just tested it: it will not work. The plugin is too old. The URL at which the plugin downloads owncloud, does not exist anymore...
